.gitignore.example

```diff
@@ -1,6 +1,8 @@
 # redaxo configuration
 # contains sensitive data and should not be stored in a git repo!
 /redaxo/data/core/config.yml
+# REDAXO installer configuration
```

Comment:
```suggestion
# redaxo installer configuration
```

scheinbar schreiben wir hier in der datei alles klein, oben ist es auch klein.

Reply:
kann es sein, dass du die beiden stellen nur als resolved markiert hast, statt die suggestion zu committen?